go/cmd/berty/main.go

```diff
@@ -169,12 +172,42 @@ func main() {
 			// protocol
 			var protocol bertyprotocol.Service
 			{
-				api, node, err := ipfsutil.NewInMemoryCoreAPI(ctx)
+
+				mardv := multiaddr.StringCast(RendezVousServer)
+				rdvpeer, err := peer.AddrInfoFromP2pAddr(mardv)
+				if err != nil {
+					return errcode.TODO.Wrap(err)
+				}
+
+				routingOpts, crouting := ipfsutil.NewTinderRouting(logger, rdvpeer.ID, false)
+				ipfsOpts := &ipfsutil.IpfsOpts{
+					Bootstrap: append(DefaultBootstrap, RendezVousServer),
```

Comment:
It's not clear why sometimes RendezVousServer is a RendezVousServer and why sometimes it's a bootstrap node, will make a better sense to duplicate the const address but use specific variable names